### PR TITLE
Aws page

### PR DESCRIPTION
**Pull request by C-STYR**
Preview [here](https://teleport-docs-next-git-aws-page-gravitational.vercel.app/use-cases/aws/)

### CODE REVIEW

File: pages/use-cases/aws/index.mdx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    <ProductBanner.Item iconWrapper title="Security &amp; Compliance" src="./assets/security-icon.svg">
+      You need to lock down who can provision & access your AWS infrastructure and keep detailed audit logs. 
+    </ProductBanner.Item>
+  </ProductBanner>
+</Section>
+
+<CenterCard 
+  src="./assets/security-icon.svg" 
+  title="Why AWS customers big and small trust Teleport" 
+  text="Our unique approach to AWS infrastructure access is not only more secure &mdash; it actually improves developer productivity.  By providing an identity-aware AWS infrastructure access solution that developers love to use, you can easily implement security and compliance without worrying about backdoors that outmoded solutions encourage."/>
```
Comment (benarent) [inline pages/use-cases/aws/index.mdx:74]: This feels very verbose..

File: pages/use-cases/aws/index.mdx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+<Section>
+  <Feedback reviews={meta.reviews} />
+</Section>
+
+<Section width="100%" bg="flatWhite">
+  <ProductBanner
+    subtitle="Get the most our of AWS Console"
```
Comment (benarent) [inline pages/use-cases/aws/index.mdx:82]:
```suggestion
    subtitle="Get the most out of AWS Management Console"
```

File: pages/use-cases/aws/index.mdx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    imgPosition="right"
+    logoSrc={startups}
+    description={
+      <>
+        With Teleport, you can separate who can provision AWS resources from who can access them.
+        We provide an identity-aware access solution for AWS Console and the AWS API that enables fine-grained, role-based access controls to prevent unauthorized creation of AWS resources.  If an SRE needs to be able to provision new AWS resources, you can easily grant temporary elevated access that automatically expires.
+      </>
+    }
+  >
+    <ProductBanner.Item title="AWS Console">
```
Comment (benarent) [inline pages/use-cases/aws/index.mdx:95]:
```suggestion
    <ProductBanner.Item title="AWS Management Console">
```

File: pages/use-cases/aws/index.mdx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+  </WorksWith>
+</Section>
+
+<EasyStart>
+      <Terminal>
+```
+# on a client
+$ tsh login --proxy=example.com
+
+# on a server
```
Comment (benarent) [inline pages/use-cases/aws/index.mdx:173]:
```suggestion
# on AWS Linux 2
```

File: pages/use-cases/aws/index.mdx [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+$ apt install teleport
```
Comment (benarent) [inline pages/use-cases/aws/index.mdx:174]:
```suggestion
$ sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
$ sudo yum install teleport
```

This command is more RHEL specific.

File: pages/use-cases/aws/index.mdx [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+  </ProductBanner>
+</Section>
+
+<Section bg="flatWhite">
+  <WorksWith title="Works with everything you have">
+    Teleport Server Access is open source and it relies on open standards such
+    as X.509 certificates, HTTPS, SAML, OpenID connect and others. Deployed as a
+    single-binary, it seamlessly integrates with the rest of your stack.
+  </WorksWith>
+</Section>
```
Comment (benarent) [inline pages/use-cases/aws/index.mdx:165]: Can we modify these icons to add CloudFormation and Terraform? ( instead of Chef & Puppet )

**Review by alexwolfe — APPROVED** [review]

**Pull request merged**